app/_data/kong_versions.yml

```diff
@@ -306,7 +306,7 @@
 - release: "3.2.x"
   version: "3.2.0"
   edition: "kubernetes-ingress-controller"
-  label: unreleased
+  latest: true
```

Comment:
I wonder if it makes sense to add 3.3 as unreleased already while we're at it. WDYT?

Reply:
Yeah, lets' do it 👍 [3736d75](https://github.com/Kong/docs.konghq.com/pull/7475/commits/3736d756e9e6bd3e2e2a010dfacde7c1edab0126)